virtcontainers/clh.go

```diff
@@ -955,6 +955,12 @@ func (clh *cloudHypervisor) LaunchClh() (string, int, error) {
 		args = append(args, "-vv")
 	}
 
+	// Disable the 'seccomp' option in clh for now.
+	// In this way, we can separate the periodic failures caused
+	// by incomplete `seccomp` filters from other failures.
+	// We will bring it back after completing the `seccomp` filter.
```

Comment:
would you mind filing an issue and mentioning it here ?

Reply:
The corresponding issue is here: https://github.com/kata-containers/runtime/issues/2899.

I will open another one to track the work on collecting `seccomp` filter lists based on kata's CI workload.

Reply:
@devimc The other issue added here: https://github.com/kata-containers/runtime/issues/2901. :)

Reply:
Thanks @likebreath. Please can you add the issue URLs into the code comments so we don't forget :)